src/sas/sascalc/doc_regen/makedocumentation.py

```diff
@@ -27,30 +28,39 @@
 DOC_LOG = USER_DOC_LOG / 'output.log'
 MAIN_DOC_SRC = USER_DOC_SRC / "source-temp"
 MAIN_BUILD_SRC = USER_DOC_SRC / "build"
+if platform.system() == 'Linux':
+    MAIN_BUILD_SRC = USER_DOC_SRC / "build" / "html"
 MAIN_PY_SRC = MAIN_DOC_SRC / "user" / "models" / "src"
 ABSOLUTE_TARGET_MAIN = Path(MAIN_DOC_SRC)
 PLUGIN_PY_SRC = Path(models.find_plugins_dir())
 
 HELP_DIRECTORY_LOCATION = MAIN_BUILD_SRC / "html"
+if platform.system() == 'Linux':
+    HELP_DIRECTORY_LOCATION = MAIN_BUILD_SRC
 RECOMPILE_DOC_LOCATION = HELP_DIRECTORY_LOCATION
 IMAGES_DIRECTORY_LOCATION = HELP_DIRECTORY_LOCATION / "_images"
 SAS_DIR = Path(sys.argv[0]).parent
 
 # Find the original documentation location, depending on where the files originate from
-if os.path.exists(SAS_DIR / "doc"):
+if platform.system() == 'Linux':
+    BASE_DIR = Path("/usr/share/doc/sasview/")
+    ORIGINAL_DOCS_SRC = BASE_DIR / "doc-source"
+    ORIGINAL_DOC_BUILD = BASE_DIR / "html"
```

Comment:
This will be wrong if you are running conda-forge version or a locally compiled version on a system which has sasview installed as a package. Similarly for other tests against 'Linux'.